controllers/webscrapeController.js

```diff
@@ -1,21 +1,106 @@
+      const currentDate = `${year}${month}${day}`;
+    
+      const url = `${espnScoreUrl}/_/date/${currentDate}`;
+      
+      await page.goto(url);
+      await page.waitForSelector(".Scoreboard__RowContainer", { timeout: 120000 });
+
+      const matchData = await page.evaluate((date) => {
+        const matchData = [];
+      
+        const matchElements = document.querySelectorAll(".Scoreboard__RowContainer");
+      
+        matchElements.forEach((container) => {
+          const matchInfo = {};
+      
+          const dateString = date.toString();
+          const matchDateObj = new Date(
+            parseInt(dateString.substring(0, 4)),
+            parseInt(dateString.substring(4, 6)) - 1,
+            parseInt(dateString.substring(6, 8))
+          );
+          const options = { year: 'numeric', month: 'long', day: 'numeric' };
+          matchInfo.matchDate = matchDateObj.toLocaleDateString('en-US', options);
+          
+          const leagueElement = container.closest(".Card")?.querySelector(".Card__Header__Title");
+          matchInfo.league = leagueElement?.textContent?.trim() ?? '';
+    
+          const homeTeamElement = container.querySelector(".ScoreboardScoreCell__Item--home .ScoreCell__TeamName");
+          matchInfo.homeTeam = homeTeamElement ? homeTeamElement.textContent.trim() : '';
+    
+          const awayTeamElement = container.querySelector(".ScoreboardScoreCell__Item--away .ScoreCell__TeamName");
+          matchInfo.awayTeam = awayTeamElement ? awayTeamElement.textContent.trim() : '';
+    
+          const homeScoreElement = container.querySelector(".ScoreboardScoreCell__Item--home .ScoreCell__Score");
+          matchInfo.homeScore = homeScoreElement ? homeScoreElement.textContent.trim() : ' ';
+    
+          const awayScoreElement = container.querySelector(".ScoreboardScoreCell__Item--away .ScoreCell__Score");
+          matchInfo.awayScore = awayScoreElement ? awayScoreElement.textContent.trim() : ' ';
+    
+          const matchStatusElement = container.querySelector(".ScoreCell__Time");
+          matchInfo.matchStatus = matchStatusElement ? matchStatusElement.textContent.trim() : '';
+    
+          const homeLogoElement = container.querySelector(".ScoreboardScoreCell__Item--home .ScoreboardScoreCell__Logo");
+          matchInfo.homeLogo = homeLogoElement ? homeLogoElement.getAttribute("src") : ' ';
+    
+          const awayLogoElement = container.querySelector(".ScoreboardScoreCell__Item--away .ScoreboardScoreCell__Logo");
+          matchInfo.awayLogo = awayLogoElement ? awayLogoElement.getAttribute("src") : ' ';
+
+          if (!matchInfo.homeLogo) {
+            matchInfo.homeLogo = 'https://www.seekpng.com/png/full/28-289657_espn-soccer-team-logo-default.png';
+          }
+        
+          if (!matchInfo.awayLogo) {
+            matchInfo.awayLogo = 'https://www.seekpng.com/png/full/28-28957_espn-soccer-team-logo-default.png';
```

Comment:
this is a bad url fyi. (put it in your browser and check, I think you miss a character)

also, all links need to be in the constants file for reusability ease

Reply:
> this is a bad url fyi. (put it in your browser and check, I think you miss a character)
> 
> also, all links need to be in the constants file for reusability ease

https://www.seekpng.com/png/full/28-289657_espn-soccer-team-logo-default.png
![28-289657_espn-soccer-team-logo-default](https://github.com/saravpereira/FieldsAndGoals-server/assets/87776690/c780f150-a20c-4d3c-abc9-67d7c605e443)